company/services/one_time_code.py

```diff
@@ -0,0 +1,66 @@
+from django.contrib.auth.models import User
+from one_day_intern.settings import EMAIL_HOST_USER
+from users.models import Company, CompanyOneTimeLinkCode
+from users.services import utils as users_utils
+from . import utils, company as company_service
+from ..exceptions.exceptions import InvalidRequestException
```

Comment:
Hey jo good job, but do you think it'll be better if you moved all the exceptions under one directory for easier maintainability and import path consistency? Let me know :D

Reply:
Hello there, Rashad. That is an excellent suggestion. I'll start using it right away.